mail_template_conditional_attachment/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_mail_template_conditional_attachment_user,mail.template.conditional.attachment.user,model_mail_template_conditional_attachment,base.group_user,1,1,1,1
```

Comment:
I think is better:
`access_mail_template_conditional_attachment_user,mail.template.conditional.attachment.user,model_mail_template_conditional_attachment,base.group_user,1,1,1,0`
`access_mail_template_conditional_attachment_system_user,mail.template.conditional.attachment.system.user,model_mail_template_conditional_attachment,base.group_system,1,1,1,1`

Reply:
Hmmm. Not sure I agree with this one.
Users are able to edit mail.templates, why wouldn't they be able to unlink conditional attachments? It's a field in the template

Reply:
I think that normally an admin user is who write the email templates, in fact I only would leave access rights of read for users of base.group_user and all access rights to users of base.group_system but in this suggestion of changes I'm following the concept that addon `mail` applies for access rights in model mail.template.

Reply:
I'd use the same permissions as per mail.template -> simple user cannot unlink

Reply:
@ivantodorovich any feedback here?

Reply:
Conditional attachments are basically a new field in mail.templates.

If simple users are able to write on mail.templates, they should be able to create/write/remove conditional attachments. Otherwise they won't be able to edit this field.